sky/data/storage.py

```diff
@@ -2021,3 +2075,392 @@ def _delete_r2_bucket(self, bucket_name: str) -> bool:
+
+        Rclone.store_rclone_config(
+            self.name,
+            Rclone.RcloneClouds.IBM,
+            self.region,  # type: ignore
+        )
+        if not bucket_region and self.sync_on_reconstruction:
+            # bucket doesn't exist
+            return self._create_cos_bucket(self.name, self.region), True
+        else:
+            # bucket exists
+            return self.s3_resource.Bucket(self.name), False
+
+    def _download_file(self, remote_path: str, local_path: str) -> None:
+        """Downloads file from remote to local on s3 bucket
+        using the boto3 API
+
+        Args:
+          remote_path: str; Remote path on S3 bucket
+          local_path: str; Local path on user's device
+        """
+        self.client.download_file(self.name, local_path, remote_path)
+
+    def mount_command(self, mount_path: str) -> str:
+        """Returns the command to mount the bucket to the mount_path.
+
+        Uses rclone to mount the bucket.
+        Source: https://github.com/rclone/rclone
+
+        Args:
+          mount_path: str; Path to mount the bucket to.
+        """
+        rclone_config_data = Rclone.get_rclone_config(
+            self.bucket.name,
+            Rclone.RcloneClouds.IBM,
+            self.region,  # type: ignore
+        )
+        # 'configure_rclone_profile' cmd stores bucket profile
+        # in rclone config file at the cluster's nodes.
+        # pylint: disable=line-too-long
+        configure_rclone_profile = (
+            f' mkdir -p ~/.config/rclone/ && echo "{rclone_config_data}">> {Rclone.RCLONE_CONFIG_PATH}'
+        )
+        install_cmd = 'rclone version >/dev/null 2>&1 || curl https://rclone.org/install.sh | sudo bash'
+        # --daemon will keep the mounting process running in the background.
+        mount_cmd = f'{install_cmd} && {configure_rclone_profile} && rclone mount {self.bucket_rclone_profile}:{self.bucket.name} {mount_path} --daemon'
+        return mounting_utils.get_mounting_command(mount_path, install_cmd,
+                                                   mount_cmd)
+
+    def _create_cos_bucket(self,
+                           bucket_name: str,
+                           region='us-east') -> StorageHandle:
+        """Creates IBM COS bucket with specific name in specific region
+
+        Args:
+          bucket_name: str; Name of bucket
+          region: str; Region name, e.g. us-east, us-south
+        Raises:
+          StorageBucketCreateError: If bucket creation fails.
+        """
+        try:
+            self.client.create_bucket(
+                Bucket=bucket_name,
+                CreateBucketConfiguration={
+                    'LocationConstraint': f'{region}-smart'
+                },
+                IBMServiceInstanceId=self.instance_id)
+            logger.info(f'{colorama.Fore.GREEN}Bucket: "{bucket_name}" '
+                        f'was created in {region}. {colorama.Style.RESET_ALL}')
```

Comment:
```suggestion
        logger.info(
            f'Created IBM COS bucket {bucket_name} in {region} '
            f'with storage class smart tier')
```
Referring to other methods, `_create_s3_bucket` or `_create_gcs_bucket`, for the log message format for consistency.

Reply:
Leaving unresolved as it is not implemented yet.